.genx/configure.js

```diff
@@ -26,11 +27,23 @@ const fs = require('fs-extra')
  */
 module.exports = async (data, utils) => {
   const {directory, pkgScope, pkgName, apiHost} = data;
-  const {editJSONFile, replaceObjectKeys, writeFileWithData} = utils;
+  const {editJSONFile, writeFileWithData} = utils;
   const prefixTarget = 'pkgScope/pkgName';
   const prefixReplacement = `${pkgScope}/${pkgName}-`;
   data.localGenId = data.appName.toUpperCase().replace("-", "_");
 
+  const writeFile = (file, data, template) => {
+    try {
+      const filePath = path.resolve(directory, file);
+      const templatePath = template ? path.resolve(directory, template) : undefined;
+      return writeFileWithData(filePath, data, templatePath);
+    } catch (e) {
+      console.error(`Error writing file ${file} using template ${template}`, e);
+      console.error('Data', data);
+      process.exit(1);
+    }
+  };
```

Comment:
As discussed, generally we want these fixes in foundation-ui, so all the seeds can use them. I'm ok for this PR to go in as is, as long as we have a follow up ticket to update the shared utils.

Basically, in this function there's better error handling and it's more user-friendly since you don't have to pass in the full path (it'll resolve it for you). We should apply these to write* methods in shared utils longer term. `directory` here comes from outer scope, which wouldn't be there in utils - we look it up from `data`, so we don't necessarily need a new param for directory - TBD.

Reply:
https://genesisglobal.atlassian.net/browse/FUI-1004